irohad/main/server_runner.hpp

```diff
@@ -14,14 +14,18 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+#include <grpc++/grpc++.h>
 #include <grpc++/server_builder.h>
 
-namespace connection {
+#ifndef CONNECTION_SERVER_RUNNER_HPP
+#define CONNECTION_SERVER_RUNNER_HPP
 
-    class ServerRunner {
-    public:
-        ServerRunner(const std::string& ip,
-                     const std::vector<grpc::Service*>& services);
-    };
+namespace server_runner {
+  void initialize(const std::string &ip, int port,
+                  const std::vector<grpc::Service *> &services);
+  void run();
```

Comment:
Think about testing. How test functionality that use server runner? If you stay only implementation, than you will need to make internet connections for each test. But good way is make mock implementation of it. Thus, other classes will use only interface of server runner.